pulsar-client-cpp/CMakeLists.txt

```diff
@@ -58,6 +58,9 @@ MESSAGE(STATUS "BUILD_TESTS:  " ${BUILD_TESTS})
 option(BUILD_PYTHON_WRAPPER "Build Pulsar Python wrapper" ON)
 MESSAGE(STATUS "BUILD_PYTHON_WRAPPER:  " ${BUILD_PYTHON_WRAPPER})
 
+option(BUILD_WIRESHARK "Build Pulsar Wireshark dissector" ON)
```

Comment:
Let's leave this disabled by default to avoid getting errors when wireshark-dev is not available.

Reply:
Thanks for your review. Addressed.